packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -13,13 +12,131 @@ import {
 } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import RootStore from 'Stores/types';
 import CryptoFiatConverter from 'Components/crypto-fiat-converter';
 import ErrorDialog from 'Components/error-dialog';
 import PercentageSelector from 'Components/percentage-selector';
 import RecentTransaction from 'Components/recent-transaction';
 import './account-transfer-form.scss';
 
-const AccountOption = ({ mt5_login_list, account, idx, is_dark_mode_on }) => {
+type TAccountOption = {
+    mt5_login_list: Array<{
+        login: string;
+        market_type: string;
+        server_info: {
+            geolocation: {
+                region: string;
+                sequence: string | number;
+            };
+        };
+    }>;
+    account: {
+        balance?: string | number;
+        currency?: string;
+        disabled?: boolean;
+        is_dxtrade?: boolean;
+        is_mt?: boolean;
+        market_type?: string;
+        nativepicker_text: string;
+        platform_icon?: string;
+        text: JSX.Element | string;
+        value?: string;
+    };
+    idx: string | number;
+    is_dark_mode_on: boolean;
+};
+
+type TAccountTransferBullet = {
+    children: React.ReactNode | JSX.Element | JSX.Element[];
+};
+
+type TAccountTransferNote = {
+    allowed_transfers_count: {
+        internal: boolean | undefined;
+        mt5: boolean | undefined;
+        dxtrade: boolean | undefined;
+    };
+    currency: string;
+    is_crypto_to_crypto_transfer: boolean;
+    is_dxtrade_allowed: boolean;
+    is_dxtrade_transfer: boolean;
+    is_mt_transfer: boolean;
+    minimum_fee: string | number;
+    transfer_fee: string | number;
+};
+
+type TAccountTransferForm = {
+    account_limits: {
+        daily_transfers?: {
+            mt5?: {
+                allowed?: boolean;
+                available?: boolean;
+            };
+            dxtrade?: {
+                allowed?: boolean;
+                available?: boolean;
+            };
+            internal?: {
+                allowed?: boolean;
+                available?: boolean;
+            };
+        };
+    };
+    account_transfer_amount: string;
+    accounts_list: Array<TAccountOption['account']>;
+    converter_from_amount: string;
+    converter_from_error: string;
+    converter_to_amount: string;
+    converter_to_error: string;
+    crypto_transactions: Array<object>;
+    error: object;
+    is_crypto: boolean;
+    is_dark_mode_on: boolean;
+    is_dxtrade_allowed: boolean;
+    minimum_fee: string;
+    mt5_login_list: TAccountOption['mt5_login_list'];
+    onChangeConverterFromAmount: () => void;
+    onChangeConverterToAmount: () => void;
+    onChangeTransferFrom: (event: object) => void;
```

Comment:
Please use `React.ChangeEvent<HTMLInputElement>` type for events, object is too general